components/mpas-ocean/bld/build-namelist

```diff
@@ -736,15 +742,19 @@ if ($OCN_ICEBERG eq 'true') {
 # (true for JRA). When atm active, we assume liquid runoff corresonds to precip
 # or snow melt so we do not remove it. In either case, the energy for melting
 # doesn't come from the ocean.
-if (($OCN_ISMF ne 'none') && ($OCN_FORCING ne 'active_atm')) {
+if (($OCN_GLC_ISMF_COUPLING ne 'none') && ($OCN_FORCING ne 'active_atm')) {
```

Comment:
@matthewhoffman, I think this is the problem.  This now presumably needs to be:
``` perl
if ((   ($OCN_GLC_ISMF_COUPLING eq 'coupler') 
     || ($OCN_GLC_ISMF_COUPLING eq 'internal_mpaso') 
     || ($OCN_GLC_ISMF_COUPLING eq 'data_mpaso') 
    ) 
    && ($OCN_FORCING ne 'active_atm')) 
{
```

Reply:
I'm testing this now.

Reply:
@xylar , thanks for hunting that down.  That indeed looks like the right solution.

Reply:
I got past this an on to a divide-by-zero.